### ISSUE TITLE
Generate training set

### ISSUE THREAD

**Issue by lsantuari** [GooglingTheCancerGenome/CNN#21]
So far, we have generated test data for the test set. We also need to generate the test data that will be use to train DeepSV.
For this, we need to have an additional set of files generated with sv-gen, like the following:

- [chr22.bam](https://github.com/GooglingTheCancerGenome/CNN/blob/iss18/data/test/chr22.bam)
- [chr22.bam.bai](https://github.com/GooglingTheCancerGenome/CNN/blob/iss18/data/test/chr22.bam.bai)
- [chr22.bedpe](https://github.com/GooglingTheCancerGenome/CNN/blob/iss18/data/test/chr22.bedpe)
- [gridss.bedpe](https://github.com/GooglingTheCancerGenome/CNN/blob/iss18/data/test/gridss.bedpe)
- [manta.bedpe](https://github.com/GooglingTheCancerGenome/CNN/blob/iss18/data/test/manta.bedpe)
- [lumpy.bedpe](https://github.com/GooglingTheCancerGenome/CNN/blob/iss18/data/test/lumpy.bedpe)
- [delly.bedpe](https://github.com/GooglingTheCancerGenome/CNN/blob/iss18/data/test/delly.bedpe)

These should contain a new set of SVs (DEL in this case) with respect to the test files that have been generated so far (and listed above). These new files should go through the same workflow that we have used so far, up to the labelling and generation of window_pair data.

Regarding the number of SVs we need for training, we probably need a few thousands. It might be a problem to generate so many with SURVIVOR in the 1 Mb region of Chr22 that we have considered so far. We can also have a smaller test, since it is used for the smoke test.

**Comment by lsantuari** [comment]
I would separate the two set of test files in different folders within the test folder, for instance `training` and `test`.

**Comment by lsantuari** [comment]
Fixed in 7ca4d97f1bfa1e3078d0c9fdd5666d3af917dc3f
